<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>382</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="ScrollingLabel" name="scrollingText_1" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="ScrollingLabel" name="scrollingText_2" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="ScrollingLabel" name="scrollingText_3" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>570</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="ScrollingLabel" name="scrollingText_4" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>570</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrollingLabel</class>
   <extends>QWidget</extends>
   <header location="global">ScrollingLabel/scrollinglabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
